tools/operator-sdk/Dockerfile

```diff
@@ -54,20 +54,20 @@ RUN GO111MODULE=on operator-sdk add api --api-version=memcached.com/v1alpha1 --k
 # We now have to pull the code generator down as a separate step
 # Add a tools.go to the example project that requires the code-generator
 # see: https://github.com/docker-library/openjdk/blob/8a23a228bda7d2edeb4132fffd2d08c1e1fcf4ac/8-jdk/Dockerfile#L28-L36
-RUN { \
-    echo 'package tools'; \
-    echo ''; \
-    echo 'import ('; \
-    echo '        _ "k8s.io/code-generator/cmd/client-gen"'; \
-    echo '        _ "k8s.io/code-generator/cmd/conversion-gen"'; \
-    echo '        _ "k8s.io/code-generator/cmd/deepcopy-gen"'; \
-    echo '        _ "k8s.io/code-generator/cmd/informer-gen"'; \
-    echo '        _ "k8s.io/code-generator/cmd/lister-gen"'; \
-    echo '        _ "k8s.io/gengo/args"'; \
-    echo '        _ "k8s.io/kube-openapi/cmd/openapi-gen"'; \
-    echo '        _ "sigs.k8s.io/controller-tools/pkg/crd"'; \
-    echo ')'; \
-    } > $GOPATH/src/github.com/example-inc/memcached-operator/tools.go
+# RUN { \
```

Comment:
None of this is needed anymore?

Reply:
I advise we remove it if we don't need it.

Reply:
oh so some of this seemed invalid because you can't import main packages, but I commented it out because I wanted to see what the tools.go they drop out of the box was